styles/css/style.css

```diff
@@ -678,6 +679,10 @@ button a {
   .footer-title {
     font-size: 14px !important;
   }
+
+  .ticket-wrap{
+    width: 100%;
+  }
 }
 
 @media(max-width:480px) {
```

Comment:
The border is visible when length is  < 768px and greater > 480px.
480px can be changes to 768px.
This issue is not introduced in this PR.